src/reducers/settings.js

```diff
@@ -0,0 +1,23 @@
+import { SET_CLIENT, SET_OFFLINE, SET_FIRST_REPLICATION, INIT_STATE } from '../actions/settings'
+
+export const initialState = {
+  offline: false,
+  firstReplication: false
+}
+
+export const settings = (state = initialState, action) => {
+  switch (action.type) {
+    case SET_CLIENT:
+      return { ...state, offline: true, client: action.client }
+    case SET_OFFLINE:
+      return { ...state, offline: action.offline }
+    case SET_FIRST_REPLICATION:
+      return { ...state, firstReplication: action.firstReplication }
+    case INIT_STATE:
+      return initialState
```

Comment:
I'm not a big fan of `INIT_STATE` action.
The reducer will take a default value from `initialState` if the state is not yet defined (`undefined` or `null`).

If we want to load a full (persisted) state, we must call a `SET_STATE` action.

But I don't know how we need to call `INIT_STATE` rather than real state changing's actions like `DO_SOMETHING`, `CHANGE_THIS`, `UPDATE_VALUE`, ...

It seems like a smell to me.